src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -224,24 +269,33 @@ void SetParameter()
+      Aux_Message( stdout, "  magnetic field profile                           = %d\n",      CCSN_Mag               );
+      Aux_Message( stdout, "  magnetic field strength                          = %13.7e\n",  CCSN_Mag_B0            );
+      Aux_Message( stdout, "  dependence of magnetic field on density          = %13.7e\n",  CCSN_Mag_np            );
+      Aux_Message( stdout, "  characteristic radius of magnetic field          = %13.7e\n",  CCSN_Mag_R0            );
 #     endif
-      Aux_Message( stdout, "  output GW signals                       = %d\n",      CCSN_GW_OUTPUT        );
-      Aux_Message( stdout, "  sampling interval of GW signals         = %13.7e\n",  CCSN_GW_DT            );
-      Aux_Message( stdout, "  mode for obtaining internal energy      = %d\n",      CCSN_Eint_Mode        );
+      Aux_Message( stdout, "  output GW signals                                = %d\n",      CCSN_GW_OUTPUT         );
+      Aux_Message( stdout, "  sampling interval of GW signals                  = %13.7e\n",  CCSN_GW_DT             );
+      Aux_Message( stdout, "  mode for obtaining internal energy               = %d\n",      CCSN_Eint_Mode         );
       if ( CCSN_Prob != Migration_Test ) {
-      Aux_Message( stdout, "  radial factor for maximum refine level  = %13.7e\n",  CCSN_MaxRefine_RadFac );
-      Aux_Message( stdout, "  scaling factor for lightbulb dt         = %13.7e\n",  CCSN_LB_TimeFac       );
-      Aux_Message( stdout, "  has core bounce occurred                = %d\n",      CCSN_Is_PostBounce    ); }
-      Aux_Message( stdout, "=============================================================================\n" );
+      Aux_Message( stdout, "  radial factor for maximum refine level           = %13.7e\n",  CCSN_MaxRefine_RadFac  );
+      Aux_Message( stdout, "  scaling factor for lightbulb dt                  = %13.7e\n",  CCSN_LB_TimeFac        );
+      Aux_Message( stdout, "  has core bounce occurred                         = %d\n",      CCSN_Is_PostBounce     ); }
+      if ( CCSN_Prob == Core_Collapse ) {
+      if ( CCSN_MaxRefine_FLAG1 ) {
+      Aux_Message( stdout, "  reduced maxmimum refinement lv 1                 = %d\n",      CCSN_MaxRefine_LV1     );
+      Aux_Message( stdout, "  central density threshold for CCSN_MaxRefine_LV1 = %13.7e\n",  CCSN_MaxRefine_Dens1   ); }
+      if ( CCSN_MaxRefine_FLAG2 ) {
+      Aux_Message( stdout, "  reduced maxmimum refinement lv 2                 = %d\n",      CCSN_MaxRefine_LV2     );
+      Aux_Message( stdout, "  central density threshold for CCSN_MaxRefine_LV2 = %13.7e\n",  CCSN_MaxRefine_Dens2   ); }
+      Aux_Message( stdout, "  central density factor for reducing dt           = %13.7e\n",  CCSN_CC_CentralDensFac );
+      Aux_Message( stdout, "  reduced dt near bounce                           = %13.7e\n",  CCSN_CC_Red_DT         ); }
+      Aux_Message( stdout, "====================================================================================\n" );
```

Comment:
If you want to print the info depending on the `CCSN_MaxRefine_FLAG*`, put the } to the correct position to make the block structure more clear.

Reply:
I think it is the correct position.

Reply:
No, it is not. The `if ( CCSN_CC_MaxRefine_Flag1/2 )` is within `if ( CCSN_Prob == Core_Collapse )`, but the corresponding `}` are at the same position as the one for `if ( CCSN_Prob != Migration_Test )`. Their position should be different to make it more readable.

Reply:
I have changed the coding style.